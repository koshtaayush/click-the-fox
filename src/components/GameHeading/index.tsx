//Library
import React from 'react'
import styled from 'styled-components'

interface Props {}

const GameHeading: React.FC<Props> = () => {
    return (
        <React.Fragment>
            <Heading>Click the Fox! Game</Heading>
        </React.Fragment>
    )
}

export default GameHeading

const Heading = styled.div`
    font-size: 2rem;
    font-weight: 500;
    margin-bottom: 2rem;
`
