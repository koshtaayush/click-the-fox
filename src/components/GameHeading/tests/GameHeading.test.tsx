import React from 'react'
import { shallow } from 'enzyme'

import GameHeading from './../index'

describe('GameHeading', () => {
    it('Should match the snapshot', () => {

        const component = shallow(
            <GameHeading />
        )

        expect(component).toMatchSnapshot()
    })

    it('Should render the proper name of component', () => {

        const component = shallow(
            <GameHeading />
        )

        expect(component.text()).toEqual('Click the Fox! Game')
    })
})
