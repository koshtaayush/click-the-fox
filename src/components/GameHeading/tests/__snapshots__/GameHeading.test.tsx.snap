// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameHeading Should match the snapshot 1`] = `
<Fragment>
  <styled.div>
    Click the Fox! Game
  </styled.div>
</Fragment>
`;
